db/hybrid_logical_vector.go

```diff
@@ -167,6 +177,9 @@ func (hlv *HybridLogicalVector) equalPreviousVectors(otherVector HybridLogicalVe
 
 // GetVersion returns the latest CAS value in the HLV for a given sourceID, if the sourceID is not present in the HLV it will return 0 CAS value
 func (hlv *HybridLogicalVector) GetVersion(sourceID string) uint64 {
+	if sourceID == "" {
+		return 0
```

Comment:
Returning zero here gives the impression that there's an entry in the HLV for `sourceID:0`.  What's the benefit of returning zero, instead of GetVersion returning (uint64, bool)?